jobOptions.mac

```diff
@@ -13,6 +13,3 @@
 #/WCSim/physics/list QGSP_BIC_HP #preferred for energies below 5 GeV
 #/WCSim/physics/list LBE # preferred for low-background experiments.  
 /WCSim/physics/list WCSim # included for backward-compatibility.  Requires jobOptions2.mac
-
-
-
```

Comment:
jobOptions.mac shows up in the diff of this pull request because there is some missing white space. Cut and paste the old jobOptions.mac here so it doesn't show up in the diff.